client/src/components/App/App.js

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
+import rootReducer from '../../ducks';
 import Landing from '../Landing/Landing';
 import './App.css';
 
+// if in development, enable redux dev tools
+const enhancer =
+  process.env.NODE_ENV === 'development'
```

Comment:
We will need to add some config files or somethin later on. good reminder of that